source/fundamentals/crud/write-operations/bulk.txt

```diff
@@ -10,28 +10,51 @@ Bulk Operations
    :depth: 2
    :class: singlecol
 
+.. meta::
+   :keywords: insert, update, replace, code example, efficiency
+
 Overview
 --------
 
 In this guide, you can learn how to use bulk operations in the
 MongoDB Java Driver. 
 
-To perform a create, replace, update, or delete operation,
-use its corresponding method. For example, to insert one document,
-update multiple documents, and delete one document in your collection,
-use the ``insertOne()``, ``updateMany()`` and ``deleteOne()`` methods.
+To perform a single create, replace, update, or delete operation, you can use
+the corresponding method. For example, to insert one document and replace one 
+document, you can use the ``insertOne()`` and ``replaceOne()`` methods. In this
+case, the ``MongoClient`` makes a call to the database for each operation. 
+
+Generally, you can reduce the number of calls to the database by using bulk write 
+operations. You can perform bulk write operations at the following levels:
+
+- :ref:`Collection Level <java-sync-coll-bulk-write>`: You can use the 
+  ``MongoCollection.bulkWrite()`` method to perform bulk write operations on a 
+  single collection. This method groups write operations together by the kind
+  of operation. For example, ``MongoCollection.bulkWrite()`` puts multiple update
+  operations in the same call, but makes two calls to the database for an insert
+  operation and a replace operation.
+
+- :ref:`Client Level <java-sync-client-bulk-write>`: When running {+mdb-server+} 
+  version 8.0 or later, you can use the ``MongoClient.bulkWrite()`` method to perform 
+  bulk write operations on multiple collections and databases in the same cluster.
+  This method groups multiple kinds of write operations into one call.
 
-The ``MongoClient`` performs these operations by making a call for each
-operation to the database. You can reduce the number of calls to the
-database to one by using bulk operations. 
+.. _java-sync-coll-bulk-write:
 
-Performing Bulk Operations
---------------------------
+Collection Bulk Write
+---------------------
 
-Bulk operations consist of a large number of write operations. To perform
-a bulk operation, pass a ``List`` of ``WriteModel`` documents to the
-``bulkWrite()`` method. A ``WriteModel`` is a model that represents any
-of the write operations. 
+Bulk operations consist of many write operations. To perform a bulk operation 
+at the collection level, pass a ``List`` of ``WriteModel`` documents to the 
+``MongoCollection.bulkWrite()`` method. A ``WriteModel`` is a model that 
+represents any of the write operations. 
+
+The ``MongoCollection.bulkWrite()`` method splits operations of different kinds into
+different batches. For example, when you pass ``DeleteOneModel``, 
+``DeleteManyModel``, and ``ReplaceOneModel`` operations to the method, the method
+splits the delete operations into one batch and the replace operation 
+into another. During this process, the client might reorder operations for 
```

Comment:
S:

```suggestion
creates two batches: one for the delete operations and one for the replace operation.
During this process, the client might reorder operations for 
```

between these two sentences, maybe mention that a batch = one database call (unless that's somewhere else on the page)